server/embed/config_logging.go

```diff
@@ -130,10 +131,17 @@ func (cfg *Config) setupLogging() error {
 
 			lvl := zap.NewAtomicLevelAt(logutil.ConvertToZapLevel(cfg.LogLevel))
 
+			var encoder zapcore.Encoder
+			if logutil.ConvertToZapFormat(cfg.LogFormat) == logutil.ConsoleLogFormat {
+				encoder = zapcore.NewConsoleEncoder(logutil.DefaultZapLoggerConfig.EncoderConfig)
+			} else {
```

Comment:
I might have missed this but do we validate the logFormat? IE `--log-format=konsole`

Reply:
I don't think so, because `ConvertToZapFormat` handles this https://github.com/etcd-io/etcd/pull/13339/files#diff-47d0d38c1df3e69bbdfb072164f8ee599204de9b3871d41d0c11b7ffed1b619cR25

Reply:
It returns `json` other than anything `console` including the empty string.

Reply:
I think we should consider returning a clear error to admin that the config was not valid vs defaulting silently.

Reply:
Indeed, you are right. I'll add validation and warning user if log-format value is incorrect.